docs/upgrade/v1-4-1-to-v1-4-3.md

```diff
@@ -1,11 +1,11 @@
 ---
 sidebar_position: 3
-sidebar_label: Upgrade from v1.4.2 to v1.4.3
-title: "Upgrade from v1.4.2 to v1.4.3"
+sidebar_label: Upgrade from v1.4.1/v1.4.2 to v1.4.3
+title: "Upgrade from v1.4.1/v1.4.2 to v1.4.3"
```

Comment:
is v141->v143 direct upgrade allowed?

add an explanation in the md that the RKE2/K8S version does not change from v142 to v143, thus v141 can upgrade to v143 directly

Reply:
Yes, upgrade from v1.4.1 is allowed because we're only doing very minor component bumps between v1.4.2 and v1.4.3:

- RKE2 v1.31.4 --> v1.31.7
- Longhorn v1.7.2 --> LH v1.7.3

I'm happy to add a note to clarify this of course.